src/main/java/org/broadinstitute/dsde/consent/ontology/resources/SwaggerResource.java

```diff
@@ -38,11 +41,11 @@ private String getSwaggerResource() {
                 if (StringUtils.isNotEmpty(p.getProperty("swagger.ui.path"))) {
                     swaggerResource = p.getProperty("swagger.ui.path");
                 } else {
-                    logger.warning("swagger.ui.path is not configured correctly");
+                    log.log(Level.WARN, "swagger.ui.path is not configured correctly");
```

Comment:
Why here is ``log.log(Level.Warn,.. `` but in StatusResource.java is ``log.warn(...``? Is the same?

Reply:
Good question ... they should be the same. For some reason, the logger here didn't have a warn method ... something isn't quite right there. Will look at it.

Reply:
OK, my bad - there is a `warn`. For some reason, my IDE wasn't compiling correctly last night. Will fix.

Reply:
Great then. That was the only thing that I came across and call my attention.

Reply:
Done!